test/testutil.h

```diff
@@ -402,4 +402,48 @@ void test_openssl_errors(void);
 extern BIO *bio_out;
 extern BIO *bio_err;
 
+/*
+ * Utilities to parse a test file.
+ */
+#define TESTMAXPAIRS        20
+
+typedef struct pair_st {
+    char *key;
+    char *value;
+} PAIR;
+
+typedef struct stanza_st {
+    const char *test_file;      /* Input file name */
+    BIO *fp;                    /* Input file */
+    char buff[10240];           /* Input buffer */
```

Comment:
Concern is that not every computer has enough RAM not to care about it. Most do, but not all. Another thing is that bug buffers belong at the end of structure. After all, [on most systems] memory is virtual and chances are that only portion will be wired to physical memory, and then putting big buffer at the end minimizes wiring...